intellij-ext/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ repositories {
 dependencies {
     implementation("io.kubernetes:client-java:16.0.0") {
         exclude(group = "org.slf4j", module = "slf4j-api")
+        exclude(group = "org.yaml", module = "snakeyaml")
```

Comment:
What's this about?

Reply:
This was a gradle warning saying the dependency was vulnerable? Something along the lines of this https://github.com/jeremylong/DependencyCheck/issues/4802 and since we dont use yaml so I excluded it